models/articles.model.js

```diff
@@ -13,21 +14,24 @@ exports.selectArticleById = (id) => {
 			else return rows
 		})
 }
-exports.selectArticles = () => {
-	return db
-		.query(
-			`SELECT articles.author, articles.title, articles.article_id, 
+exports.selectArticles = (topic) => {
+	const values = []
+	const conditions = []
```

Comment:
What is this for? It is not being used. 🙂

Reply:
ohh that makes sense, those topics filter comes from topics list, needs to be verified from topics table right.

Reply:
yeah conditions var is not being used,I 
 will delete it